src/deploy.sh

```diff
@@ -476,6 +485,30 @@ function modules_install_to()
   cmd_manager "$flag" "$cmd"
 }
 
+# This function will copy and install the compiled dtb files
+# @target Where should the dtb files be installed (1-vm, 2-local or 3-remote)
+function dtb_install()
+{
+  local target="${1:configuration[default_deploy_target]}"
+  local architecture="${configurations[arch]}"
+  # the rsync_params variable is passed directly to rsync (also used in cp2remote) and
+  # used to copy only the dtb files and ignore everything else under the dts folder
+  local rsync_params='--include="*/" --include="*.dtb*" --exclude="*"'
+
+  if [[ -d "arch/$architecture/boot/dts/" ]]; then
+    case "$target" in
+      1 | 3) # VM_TARGET or REMOTE_TARGET
+        cp2remote "arch/$architecture/boot/dts/" "/boot/" '' "$rsync_params"
+        ;;
+      2) # LOCAL_TARGET
+        cmd_manager '' "rsync -LrlptD $rsync_params arch/$architecture/boot/dts/ /boot/ --rsync-path='sudo rsync'"
```

Comment:
Do we really need this complicated command? I mean, this is a local deploy, a simple copy should be enough.

Reply:
yes, unless we want to break this into several comands (which personally I find more brittle and error prone)